.circleci/config.yml

```
@@ -0,0 +1,62 @@
+version: 2.1
+
+executors:
+  go-executor:
+    docker:
+      - image: okteto/golang-ci:2.6.2
+
+jobs:
+  setup:
+    executor: go-executor
+    steps:
+      - checkout
+      - run:
+          name: Install Go Modules
+          command: go mod download
+
+  build:
+    executor: go-executor
+    steps:
+      - checkout
+      - run:
+          name: Build the project
+          command: go build -o bin/okteto-cli -ldflags="-s -w" ./cmd/main.go
+      - persist_to_workspace:
+          root: .
+          paths:
+            - bin
+
+  test:
+    executor: go-executor
+    steps:
+      - checkout
+      - run:
+          name: Run Unit Tests
+          command: go test ./... -v
+
+  docker-build-and-cve-scan:
+    executor: go-executor
+    steps:
+      - setup_remote_docker:
+          version: 20.10.7
+      - checkout
+      - run:
+          name: Build Docker Image
+          command: docker build -t okteto-cli:latest .
```

```suggestion
          command: docker build -t okteto-test-action:latest .
```

I think `okteto-cli` is misleading as it seems you are build the Okteto CLI